plugins/woocommerce/src/Database/Migrations/CustomOrderTable/CLIRunner.php

```diff
@@ -368,8 +378,13 @@ public function verify_cot_data( $args = array(), $assoc_args = array() ) {
 		if ( 0 === count( $failed_ids ) ) {
 			return WP_CLI::success(
 				sprintf(
-					/* Translators: %1$d is the number of migrated orders and %2$f is time taken */
-					_n( '%1$d order was verified, in %2$f seconds', '%1$d orders were verified in %2$f seconds', $processed, 'woocommerce' ),
+					/* Translators: %1$d is the number of migrated orders and %2$d is time taken. */
+					_n(
+						'%1$d order was verified in %2$d seconds.',
+						'%1$d orders were verified in %2$d seconds.',
+						$processed,
+						'woocommerce'
+					),
```

Comment:
- Adjusted the translator comment.
- Changed the format of the execution time in seconds from float to integer.
- Remove obsolete comma in singular string.